### ISSUE TITLE
Check for repeatable question group observations being array and not object

### ISSUE THREAD

**Issue by petmongrels** [avniproject/avni-server#593]
There was one observation in Lahi which was an object. This causes the ETL job to fail. ETL failure is downstream and it can result in bad data getting created that needs to be handled via support. We should have a check and correction of data when such data is being pushed to the server.

**Comment by vinayvenu** [comment]
@petmongrels confirming, are you suggesting making this an array by default in avni-server and not worry about the source of the error on avni-client/avni-webapp? 

One problem here could be that the form was made repeatable after some data already came in. This can cause issues in a similar pattern. In which case, the fix would probably have to be somewhere else.

**Comment by petmongrels** [comment]
yes. because throwing sync problem will require fixing problem on the mobile app which is going to be difficult. It is possible that this is due to bug in Avni client.

for the second point - I just asked Nupoor to raise an issue to now allow changing repeatable value after saving on form element.

but we will have to do both.
